module/apmgorm/apmgorm_test.go

```diff
@@ -148,30 +151,63 @@ func TestWithContextNonSampled(t *testing.T) {
 }
 
 func TestCaptureErrors(t *testing.T) {
-	db, err := apmgorm.Open("sqlite3", ":memory:")
-	require.NoError(t, err)
-	defer db.Close()
+	t.Run("sqlite3", func(t *testing.T) {
+		db, err := apmgorm.Open("sqlite3", ":memory:")
+		require.NoError(t, err)
+		testCaptureErrors(t, db)
+		defer db.Close()
```

Comment:
`defer db.Close()` needs to be just above `testCaptureErrors(t, db)`, in case testCaptureErrors panics

Reply:
Fixed :D